src/ngraph/ngraph_emitter.cc

```diff
@@ -62,17 +62,35 @@ void Emitter::ClearOpMap() {
  * based index), where
  * negative values means indexing from the right.
  */
+inline size_t transform_axis(int axis, int shape_size) {
+  assert(abs(axis) <= shape_size);
+  // convert negative axis index to postive (counting from right per mxnet
+  // convention)
+  return axis < 0 ? shape_size + axis : axis;
+}
+
 inline size_t get_default_transformed_axis(const NodePtr& node,
                                            const std::string& key,
                                            const int default_val,
                                            const int shape_size) {
-  int axis = get_default(node, key, default_val);
-  assert(abs(axis) <= shape_size);
-  // convert negative axis index to postive (counting from right per mxnet
-  // convention)
-  size_t transformed_axis = axis < 0 ? shape_size + axis : axis;
+  return transform_axis(get_default(node, key, default_val), shape_size);
+}
 
-  return transformed_axis;
+inline std::vector<size_t> get_default_transformed_axis(
+    const NodePtr& node, const std::string& key,
+    const ngraph::AxisVector& default_val, const ngraph::Shape shape) {
+  std::vector<int> values;
```

Comment:
We are doing this because `AxisVector` is `vector<size_t>` and we need `vector<int>` for `get_default`?